README.md

```diff
@@ -128,12 +125,96 @@ Invokes all Unit tests from schema/package "hr_test" with two reporters:
 * ut_coverage_html_reporter - will report only on database objects that are mapping to file structure from "source" folder and save output to file "coverage.html"
 
 ```
-utplsql run hr/hr@xe
+> utplsql run hr/hr@xe
 ```
 
 Invokes all unit test suites from schema "hr". Results are displayed to screen using default ut_documentation_reporter.
 
-#### Enabling Color Outputs on Windows
+### info
+`utplsql info [<ConnectionURL>]`
+
+```
+<ConnectionURL>     - accepted formats:
+                        <user>/<password>@//<host>[:<port>]/<service>
+                        <user>/<password>@<host>:<port>:<SID> 
+                        <user>/<password>@<TNSName> 
+                      To connect using TNS, you need to have the ORACLE_HOME environment variable set.
+                      The file tnsnames.ora must exist in path %ORACLE_HOME%/network/admin
+                      The file tnsnames.ora must contain valid TNS entries. 
+```
+
+#### Examples
+
+```
+> utplsql info
+
+cli 3.1.1-SNAPSHOT.local
+utPLSQL-java-api 3.1.1-SNAPSHOT.123
+```
+```
+> utplsql info app/app@localhost:1521/ORCLPDB1
+
+cli 3.1.1-SNAPSHOT.local
+utPLSQL-java-api 3.1.1-SNAPSHOT.123
+utPLSQL 3.1.2.1913
+```
+
+### reporters
+`utplsql info <ConnectionURL>`
+
+```
+<ConnectionURL>     - accepted formats:
+                        <user>/<password>@//<host>[:<port>]/<service>
+                        <user>/<password>@<host>:<port>:<SID> 
+                        <user>/<password>@<TNSName> 
+                      To connect using TNS, you need to have the ORACLE_HOME environment variable set.
+                      The file tnsnames.ora must exist in path %ORACLE_HOME%/network/admin
+                      The file tnsnames.ora must contain valid TNS entries. 
+```
+
+#### Examples
+```
+> utplsql reporters app/app@localhost:1521/ORCLPDB1
+
+UT_COVERAGE_COBERTURA_REPORTER (SQL): Generates a Cobertura coverage report providing information on code coverage with line numbers.
```

Comment:
What does (SQL) / (SQL_with_Java) mean? 
I find it confusing. 
Do we need some extra actions for one or another?

Reply:
I guess the concept needs more explanation, still struggling of how to do so.
With 3.1.0 ReporterFactory in cli/java-api you can have 
- Reporters solely implemented on the database (most core reporters) -> SQL
- Reporters solely implemented on the client. You could for example write a Reporter which uses ut_documentation_reporter internally but modifies the output excessively and is registered via a different name (so not overwriting ut_documentation_reporter but just using it internally). -> JAVA
- Reporters who are implemented on the database but also have some kind of addition on the client. Best example is the ut_coverage_html_reporter: It's basically the database implemented reporter but has this client-side addition that the assets are written to a specific folder. -> SQL_WITH_JAVA

Reply:
I don't think users can implement reporters in cli.
We (as dev team) would have to do that.
I would refrain from implementing reporters (based on existing PLSQL reporters) in java.

Can you remove that info?
I'd like to avoid questions/confusinon from users.

If needed, we can add this info to documentation.
The comment seems a good explanation.